package.json

```diff
@@ -283,5 +284,121 @@
       "atom",
       "snapshotResult"
     ]
+  },
+  "versionTracker": {
+    "successfulBuilds": {
+      "1.53.0-dev": [
+        {
+          "platform": "darwin",
+          "executables": {
+            "node": "v10.20.1",
+            "npm": "6.14.6",
+            "xcode": "Xcode 9.2\nBuild version 9C40b",
+            "clang": "Apple LLVM version 9.0.0 (clang-900.0.39.2)\nTarget: x86_64-apple-darwin16.7.0\nThread model: posix\nInstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin",
+            "visual studio": null,
+            "apm": "apm  2.5.0\nnpm  6.14.5\nnode 10.20.1 x64\natom 1.51.0-dev-4a9d56f52\npython 3.7.7\ngit 2.27.0",
+            "powershell": "PowerShell 6.2.3",
+            "python": "Python 3.7.7",
+            "python2": "Python 2.7.17",
+            "python3": "Python 3.7.7",
+            "zsh": "zsh 5.8 (x86_64-apple-darwin16.7.0)",
+            "bash": "GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin16)\nCopyright (C) 2007 Free Software Foundation, Inc.",
+            "git": "git version 2.27.0"
+          }
+        }
+      ]
+    },
+    "track": [
+      {
+        "name": "node",
+        "versionCommand": [
+          "node",
+          "--version"
+        ]
+      },
+      {
+        "name": "npm",
+        "versionCommand": [
+          "npm",
+          "-v"
+        ]
+      },
+      {
+        "name": "xcode",
+        "versionCommand": [
+          "/usr/bin/xcodebuild",
+          "-version"
+        ]
+      },
+      {
+        "name": "clang",
+        "versionCommand": [
+          "clang",
+          "--version"
+        ]
+      },
+      {
+        "name": "visual studio",
+        "versionCommand": [
+          "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\vswhere.exe"
```

Comment:
`vswhere` does not give the version of the installed visual studio.

Reply:
Do you know of a global command for Visual Studio?

Reply:
Yes, but it is not that straight forward.
```ps1
vswhere -products * -find **/Hostx64/x64/*
```
gives the list of all installed visual studio installations. Then we should find `2017` from the given path.

Reply:
OK here is a one liner:
```ps1
❯ (vswhere -products * -find **/Hostx64/x64/* | select-string '(\\MSVC\\)(.*)(\\bin)').matches.groups[2].value
14.16.27023
```
gives

Reply:
Thanks! that helps me out a ton.

Reply:
Do you install `vswhere` in your script? Since this is not available on Windows by default.